src/libraries/System.Text.Json/Common/JsonKnownNamingPolicy.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// The <see cref="Json.JsonNamingPolicy"/> to be used at run-time.
+    /// </summary>
+#if BUILDING_SOURCE_GENERATOR
+    internal
+#else
+    public
+#endif
+    enum JsonKnownNamingPolicy
+    {
+        /// <summary>
+        /// Specifies that JSON property names should not be converted.
+        /// </summary>
+        Unspecified = 0,
+
+        /// <summary>
+        /// Speciies that the built-in <see cref="Json.JsonNamingPolicy.CamelCase"/> be used to convert JSON property names.
+        /// </summary>
+        BuiltInCamelCase = 1
```

Comment:
Question on naming: why "BuiltIn" -- would there ever be a value here that's not "built in"? i.e. how would external naming policies fit in here?

Reply:
We would only add "built in" policies to this enum, so we could drop "BuiltIn" from the name here and take that back to API review. Taking a note to do this offline but will leave it as-is this PR.